test/scripts/kind_cluster.yaml

```diff
@@ -20,9 +20,6 @@ nodes:
   - containerPort: 8080 # keycloak
     hostPort: 8080
     protocol: TCP
-  - containerPort: 8443 # keycloak
-    hostPort: 8443
-    protocol: TCP
   - containerPort: 5432 # postgresql DB
```

Comment:
Do we need the DB port to be exposed by default?

Reply:
this is actually a development KIND cluster setup so I'd keep the DB port exposed by default.
However I've removed it from the example KIND cluster setup in `/deploy/kind.yaml`

Reply:
it is used for testing the db layer in isolation.